### PR TITLE
Practical 08 feedback

### PR DESCRIPTION
**Pull request by tananhnguyen**
Please give me feedback on mycode
@Albert-Alvaro
@CynthiaJCU
@ArthurMY
@AdityaVijay1
@Sathvika-7276

### CODE REVIEW

**Comment by tananhnguyen** [comment]
Please comment on my code

File: Practical 8/convert_miles_km.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        """check for valid input value"""
+        try:
+            input_value = float(self.root.ids.text_input.text)
+        except ValueError:
+            input_value = 0.0
+        return input_value
+
+
+ConvertMilesToKmApp().run()
+
```
Comment (Sathvika-7284) [inline Practical 8/convert_miles_km.py:45]: looks good!

File: Practical 8/dynamic_labels.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        return self.root
+
+    def create_name_widgets(self):
+        """Create buttons from data and add them to the GUI."""
+        for name in self.names:
+            label = Label(text=name)
+            self.root.ids.main.add_widget(label)
+
+
+DynamicWidget().run()
```
Comment (Sathvika-7284) [inline Practical 8/dynamic_labels.py:28]: nice work!

File: Practical 8/convert_miles_km.kv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            orientation: 'vertical'
+            Button:
+                text: 'Up'
+                id: up_button
+                on_press: app.handle_increment(1)
+            Button:
+                text: 'Down'
+                id: down_button
+                on_press: app.handle_increment(-1)
+#    Button:
```
Comment (Sathvika-7284) [inline Practical 8/convert_miles_km.kv:19]: convert miles to km button is missing

**Pull request merged**